public/views/index-template.html

```diff
@@ -295,6 +295,27 @@
       [[end]]
     </script>
 
+[[if .TrustedTypes]]
+
+<script nonce="[[.Nonce]]">
+  if (window.trustedTypes && trustedTypes.createPolicy) { // check if browser supports Trusted Types
```

Comment:
is it best to do things here?  or can we do it in the app init function?

https://github.com/grafana/grafana/blob/main/public/app/app.ts#L111

Reply:
My motivation in asking the question is to get complex js code out of the index template and into the javascript build :)

In particular, this should not depend on `purify.min.js` directly :)

Reply:
Yes, `purify.min.js` was added directly into the index template to get the prototype working. I will try to refactor the code to depend on our DOMPurify. 

Regarding app init function: if I try to create a default TT policy (`trustedTypes.createPolicy()`), it will be created *after* jQuery and Angular has loaded, so there seems to be some race condition. Check the screenshot where you can see that we get 7 violations (76 without a TT policy). 

So, is there another place we can put the policy creation when all code has loaded? 

![image](https://user-images.githubusercontent.com/114284895/226607407-584e7e9f-0c85-4e5a-b689-4215649c0671.png)